### DISCUSSION TITLE
对read_bytes里的content解密的话播放器无法播放，浏览器正常

### DISCUSSION THREAD

**Discussion by ykxVK8yL5L** [messense/aliyundrive-webdav#846] [Ideas]
您好，目前fork了一份您的aliyundrive-webdav，做了一些修改：在read_byres方法里对download的结果进行aes-128-ctr的解密。浏览器播放视频时一切正常，调用第三方播放器的话有时间进度条没有画面和声音.不知改如何解决，跪求指导🙏 这是项目地址：https://github.com/ykxVK8yL5L/FastWebdav/tree/encrypt    
 注意请选择encrypt分支.
其中aesctr.rs为解密算法目前浏览器播放视频一切正常.

**Comment by ykxVK8yL5L** [comment]
已经解决

**Comment by MXYZR** [comment]
这个现在用不了了吗？

**Reply by ykxVK8yL5L** [reply]
你说的哪个项目？官方这个的话  从阿里云开始限制我就没再用过了
